.github/workflows/release_nightly.yml

```diff
@@ -14,13 +14,14 @@ jobs:
     runs-on: ubuntu-22.04
     outputs:
       is_active: ${{ steps.activity.outputs.is_active }}
-      date: ${{ steps.current_time_underscores.outputs.formattedTime }}
+      date: ${{ steps.current_time_dashes.outputs.formattedTime }}
       upload_url: ${{ steps.create_release.outputs.upload_url }}
-      package_prefix: ruffle-nightly-${{ steps.current_time_underscores.outputs.formattedTime }}
+      package_prefix: Ruffle_Nightly_${{ steps.current_time_dashes.outputs.formattedTime }}
```

Comment:
The problem is: These are not technical filenames. These are the filenames every user will see after downloading the package, and some will even decide the version to download depending on it (when using GitHub releases).
Therefore, they have to be as easily readable and understandable as possible. E.g. when downloading Firefox, the name is `Firefox Installer.exe` or `Firefox 129.0.1.dmg` - easy to read and understand for every user.
Dashes are in the same height as letters and blend under while underscores create a visible gap (and are longer). `Re-ad_Th-is_Sent-ence_He-re` is much easier to read than `re_ad-th_is-sen_tence-he_re` - the gaps are in the wrong place.

Reply:
I would argue they _should_ be technical filenames. GitHub is just not friendly for non-technical users, it was never its goal. That's why the most basic (and some advanced) downloads are linked from the Ruffle website. Users are redirected to GitHub not because GH's user-friendly, but because it's easy to do so. GitHub does not provide any way of differentiating assets based on OS, platform, or architecture. It does not have any way of detecting which OS the user has in order to show what to download. The releases page is just not meant for that, and I don't believe we should force it to be a bit user-friendly by playing with asset names. The user which lands on releases already is required to have technical knowledge about their system and in general.

When it comes to filenames after downloading the package (from the website), I believe it's a wrong place to change that. If we decide that in fact these filenames are non-user-friendly and we want to change them, we should change them on the website, not here (and I suspect it would currently be a bit difficult as we're redirecting to GH).

Reply:
I don't really see any reason for keeping worse filenames. Maybe you're right that it doesn't matter than much to some people as I think it does, but it's still just better than doing nothing, even if it's not _important_.
The filenames should be consistent across the board. You're right that if the website has different filenames, we should change them there as well.
But also, I would argue it doesn't matter why users are redirected to Github. As long as they _are_, the filenames on GitHub matter. And it doesn't hurt to at least try and make it easier for less tech-savvy users to still download Ruffle when they are redirected.

Reply:
These filenames are used by the website to figure out which file is which, and present the relevant stuff to users. If we want friendly names it should be done via the website, github names are indeed technical and these should be considered closer to release artifacts instead of user deliverables.